backend/wait_for_db.py

```
@@ -0,0 +1,21 @@
+from django.db.utils import OperationalError
+
+def wait_for_db():
+    """Wait until MySQL is available."""
+    db_conn = None
+    while db_conn is None:
+        try:
+            print("Waiting for database...")
+            db_conn = MySQLdb.connect(
+                host='db',  # The service name defined in your docker-compose.yml
+                user='mhpcoder',
+                password='justkidding12!@',
```

Ruff (0.8.2)

Possible hardcoded password assigned to argument: "password"

(S106)